### PR TITLE
Feature/bc 307 transformation hpi schul cloud in d bildungscloud

### PR DESCRIPTION
**Pull request by monaweitzenberg**
theme and content transformation HPI Schul-Cloud into dBildungscloud

Links to Tickets or other pull requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-307

Changes
Platform

adjust primary and secondary color
exchange Logo
exchange default-file: instance_cfg.yml (line 3-6) title and contact
delete the PartnerTeam, About pages
clean up the footer
adjust imprint
Landing-Page

adjust primary and secondary color
exchange Logo
delete sections: partner; newsletter

Data Security details on Confluence
Deployment
New Repos, NPM packages or vendor scripts
Screenshots of UI changes
Approval for review
 QA: In addition to review, the code has been manually tested (if manual testing is possible)
 All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

Link to Definition of Done
More and detailed information on the definition of done can be found on Confluence

### CODE REVIEW

File: src/themes/open/components/templates/impressum.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<template>
```
Comment (pkleybolte) [inline src/themes/open/components/templates/impressum.vue:1]: the impressum of open was moved to int and this one was created new?
I don't understand why.
This one is all about HPI. Shouldn't it be about Dataport as well?

Comment (monaweitzenberg) [reply src/themes/open/components/templates/impressum.vue:1]: The imprint pages of 'Open' and 'International' should be have the content of the live pages right now. Pls check.
https://international.hpi-schul-cloud.de/imprint
https://open.hpi-schul-cloud.de/imprint

**Comment by monaweitzenberg** [comment]
@schul-cloud-bot update

**Review by pkleybolte — DISMISSED** [review]

**Review by pkleybolte — DISMISSED** [review]

**Comment by monaweitzenberg** [comment]
@schul-cloud-bot update

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Comment by pkleybolte** [comment]
@schul-cloud-bot update screenshots

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**